src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-router.ts

```diff
@@ -1,21 +1,22 @@
 import { Router } from "express";
-import { JiraContextJwtTokenMiddleware, JiraJwtTokenMiddleware } from "middleware/jira-jwt-middleware";
 import { GithubServerAppMiddleware } from "middleware/github-server-app-middleware";
 import { csrfMiddleware } from "middleware/csrf-middleware";
 import { JiraConnectEnterpriseAppDelete } from "routes/jira/connect/enterprise/app/jira-connect-enterprise-app-delete";
 import { JiraConnectEnterpriseAppPut } from "routes/jira/connect/enterprise/app/jira-connect-enterprise-app-put";
 import { JiraConnectEnterpriseAppPost } from "routes/jira/connect/enterprise/app/jira-connect-enterprise-app-post";
 import { JiraConnectEnterpriseAppCreateOrEdit } from "routes/jira/connect/enterprise/app/jira-connect-enterprise-app-create-or-edit";
+import { JiraContextJwtTokenMiddleware, JiraJwtTokenMiddleware } from "~/src/middleware/jira-jwt-middleware";
+import { jiraSymmetricJwtMiddleware } from "~/src/middleware/jira-symmetric-jwt-middleware";
 
 export const JiraConnectEnterpriseAppRouter = Router();
 
-JiraConnectEnterpriseAppRouter.post("/", JiraContextJwtTokenMiddleware, JiraConnectEnterpriseAppPost);
+JiraConnectEnterpriseAppRouter.post("/", JiraContextJwtTokenMiddleware, jiraSymmetricJwtMiddleware, JiraConnectEnterpriseAppPost);
 
 const routerWithUUID = Router({ mergeParams: true });
 JiraConnectEnterpriseAppRouter.use("/:uuid", routerWithUUID);
 
 routerWithUUID.use(GithubServerAppMiddleware);
```

Comment:
**IMPORTANT**
This is missing the new middleware (for `res.locals.jiraHost`)

Reply:
Tbh, I think we need another refactoring (not this PR).
I spend quite a bit of time and look carefully to find this bug. Feels like:
1. We either missing tests (which is the case, but hard to fix)
2. And this is example of why nesting router doesn't help code readability from security point of view.

Reply:
Added `jiraSymmetricJwtMiddleware` for `JiraConnectEnterpriseAppRouter` router